src/mono/mono.proj

```diff
@@ -65,7 +65,7 @@
   </Target>
 
   <Target Name="BuildMonoRuntimeUnix" Condition="'$(OS)' != 'Windows_NT'" DependsOnTargets="ConfigureMonoRuntimeUnix">
-    <Exec Command="make -j8" IgnoreStandardErrorWarningFormat="true" WorkingDirectory="$(MonoObjDir)" />
+    <Exec Command="bash -c 'make -j%24(nproc)'" IgnoreStandardErrorWarningFormat="true" WorkingDirectory="$(MonoObjDir)" />
```

Comment:
I don't think `$(nproc)` works on all platforms, or at least not without installing additional packages.

This should be more portable:

```suggestion
    <Exec Command="make -j$([System.Environment]::ProcessorCount)" IgnoreStandardErrorWarningFormat="true" WorkingDirectory="$(MonoObjDir)" />
```

Reply:
I only tested it on macOS and Debian 10, and you are probably right, because the `nproc` binary lives in `/usr/local` on my macOS machine.

I've tested your suggestion, works great. Thank you.